src/main/java/user/votehelpful/HelpfulInteractor.java

```diff
@@ -5,13 +5,21 @@
 import service.dao.IReviewDAO;
 import user.votehelpful.exceptions.ReviewNotFoundException;
 
+/** The votehelpful use case interactor that calls the create method from the
+ * IHelpfulInputBoundary input boundary. When initialized, takes in an object that
+ * implements IReviewDAO to access the database through.
+ */
 public class HelpfulInteractor implements IHelpfulInputBoundary {
     private final IReviewDAO reviewDAO;
 
     public HelpfulInteractor(IReviewDAO reviewDAO) {
         this.reviewDAO = reviewDAO;
     }
 
+    /** Increases the count of helpful or unhelpful by 1.
+     * @param requestModel the request model
+     * @return a response model to be sent back to the client
+     */
```

Comment:
same as above, add an explanation about the try catch exception

Reply:
Same explanation as above.